packages/asc-ui/src/components/ShowMoreShowLess/ShowMoreShowLessStyle.ts

```diff
@@ -5,6 +5,7 @@ export type ShowMoreShowLessProps = {
   maxHeight?: string
   showMoreText?: string
   showLessText?: string
+  gradientBackground?: string
```

Comment:
This seems an unrelated change, I think it should be removed.

Reply:
Will open a separate PR for this. excuse my laziness 😉